src/components/oculus-touch-controls.js

```diff
@@ -320,20 +357,121 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
     });
   },
 
+  applyOffset: function (model) {
+    model.position.copy(this.displayModel[this.data.hand].modelPivotOffset);
+    model.rotation.copy(this.displayModel[this.data.hand].modelPivotRotation);
+  },
+
+  multiMeshFix: function (button) {
+    if (!this.buttonMeshes[button]) return;
+    this.buttonMeshes[button].traverse((node) => {
```

Comment:
no arrow function for consistency

Reply:
use regular function and `var self = this;` pattern

Reply:
will do